cmd/bblfshd/main.go

```diff
@@ -115,10 +117,15 @@ func main() {
 		os.Exit(1)
 	}
 
-	parsedBuild, err := time.Parse("2006-01-02T15:04:05-0700", build)
+	parsedBuild, err := time.Parse(daemon.BuildDateFormat, build)
 	if err != nil {
-		logrus.Errorf("wrong date format for build: %s", err)
-		os.Exit(1)
+		logrus.Errorf("wrong date format for this build: %s", err)
```

Comment:
This error should be in `else` for `undefined`.

Reply:
I like the idea of this being logged in any case, but can move to the `os.Exit()` branch, if that is what you are asking for.

Reply:
Yes, I think exit is better. Running dev version shouldn't lead to errors like this :)